env/WCOSS2.env

```
@@ -358,6 +358,11 @@ elif [[ "${step}" = "fit2obs" ]]; then
     export NTHREADS_FIT2OBS=${NTHREADS1}
     export MPIRUN="${launcher} -np ${ntasks}"
 
+elif [[ "${step}" = "gen_control_ic" ]]; then
+
+    export NTHREADS_CHGRES=${NTHREADS_CHGRES:-1}
+    export APRUN_CHGRES=${APRUN_CHGRES:-"mpiexec -l -n ${ntasks} -ppn ${tasks_per_node} --cpu-bind core"}
```

```suggestion
    export APRUN_CHGRES="${launcher} -n ${ntasks} -ppn ${tasks_per_node} --cpu-bind core"
```
Please look here and see why `-np ${tasks}` is not appropriate.